docs/components/modeler/forms/form-element-library/forms-element-library-html.md

```diff
@@ -0,0 +1,66 @@
+---
+id: forms-element-library-html
+title: HTML view
+description: A form element to display HTML content.
+---
+
+A flexible display component designed to quickly render HTML content for the user.
+
+![Form HTML Symbol](/img/form-icons/form-html.svg)
+
+## Configurable properties
+
+- **Content**: This property accepts HTML content. It can be defined as an [expression](../../feel/language-guide/feel-expressions-introduction.md), using [templating syntax](../configuration/forms-config-templating-syntax.md) or as plaintext HTML. Javascript and `<style>` tags are purposefully sanitized out for the sake of security, inline styles are however perfectly valid.
+- **Hide if**: [Expression](../../feel/language-guide/feel-expressions-introduction.md) to conditionally hide the HTML content.
+- **Columns**: Space the field will use inside its row. The **Auto** means it will automatically adjust to available space in the row. Read more about the underlying grid layout in the [Carbon Grid documentation](https://carbondesignsystem.com/guidelines/2x-grid/overview).
+
+## HTML configurations
+
+These examples demonstrate how to use HTML to display content. JavaScript and `<style>` tags are entirely filtered out of the rendered output, which happens after any templating or expression evaluation. This allows this component to be very flexible in terms of display use cases without exposing any security risks such as XSS. Styling in-line via style attributes is allowed.
+
+**Basic HTML**:
+
+```
+<div>
+  <h1>Welcome to Our Site</h1>
+  <p>This is a paragraph with some <strong>bold text</strong> and <em>italic text</em>.</p>
+  <a href="https://example.com">Click here</a> to visit our page.
+</div>
+```
+
+**HTML with inline styling**:
+
+```
+<div style="background-color: lightblue; padding: 10px;">
+  <h2 style="color: navy;">Styling Example</h2>
+  <p style="font-size: 14px;">This paragraph is styled with inline CSS.</p>
+</div>
+```
+
+**List and Images**:
+
+```
+<ul>
+  <li>First Item</li>
+  <li>Second Item</li>
+</ul>
+
+<img src="https://someurl.com/image.png" alt="Descriptive Image Text">
+
+```
+
+**Templating Notation**
```

Comment:
Nitpick, but we can make these sentence-case, so "Templating notation" for example.